mne/io/base.py

```diff
@@ -2004,9 +2024,8 @@ def append(self, raws, preload=None):
         self.annotations = annotations
         for edge_samp in edge_samps:
             onset = _sync_onset(self, (edge_samp) / self.info['sfreq'], True)
-            self.annotations.append(onset, (1. / self.info['sfreq']),
-                                    'BAD boundary')
-
+            self.annotations.append(onset, 0., 'BAD boundary')
```

Comment:
why both 'BAD' and 'edge'? If an epoch starts perfectly where the segment starts - this epoch would be rejected during epoching due to 'BAD' event, right?